code/CO2_monitor.ino

```diff
@@ -334,6 +369,20 @@ void loop(){
       co2_web         = co2_new;
       temperature_web = temperature_new;
       humidity_web    = humidity_new;
+#if WIFI_MQTT
+      client.connect(deviceName);
```

Comment:
Following my comment/suggestion from above this might need to be changed to
`mqttClient.connect(deviceName);`